tests/integrations/test_flask.py

```diff
@@ -211,3 +212,21 @@ def test():
                 "framework": "Flask"
             }
         })
+
+    def test_appends_framework_version(self):
+        app = Flask("bugsnag")
+
+        @app.route("/hello")
+        def hello():
+            raise SentinelError("oops")
+
+        handle_exceptions(app)
+        app.test_client().get('/hello')
+
+        self.assertEqual(len(self.server.received), 1)
+
+        payload = self.server.received[0]['json_body']
+        device_data = payload['events'][0]['device']
+
+        self.assertEqual(device_data['runtimeVersions']['flask'],
+                         flask.__version__)
```

Comment:
Perhaps it would be a good idea to test a bit more details about the version set here, the test lacks protection against changes in flask regarding the `__version__` and gives a false sense of security.

For example, if in the next major version of Flask the API of `__version__` changes to be an array of numbers or an object (example: `__version__ = {'major': 18, 'minor': 0, 'patch': 0}`) this test would continue to pass after upgrading flask. That means anyone in the future going to ensure that the integration works with a new version of Flask (the same applies to the other integrations) may wrongly expect that the code is compatible with the newer version. Which isn't correct, it would unexpectedly change the format of the version sent to the server.

It may be a good idea to also test that `device_data['runtimeVersions']['flask']` matches a regex like `\d+\.\d+\.\d+` which ensures it is a string of the expected format?

Reply:
Good point - I'll amend.